litmus-portal/namespaced-k8s-template.yml

```diff
@@ -697,3 +697,18 @@ spec:
       targetPort: 27017
   selector:
     component: database
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: mongo
+  name: mongo-headless-service
+  namespace: litmus
```

Comment:
Namespace not required for namespaced-mode

Reply:
done